tomox/order_processor.go

```diff
@@ -541,3 +543,67 @@ func DoSettleBalance(coinbase common.Address, takerOrder, makerOrder *tomox_stat
 	tomox_state.SetTokenBalance(makerExOwner, newMakerFee, makerOrder.QuoteToken, statedb)
 	return nil
 }
+
+func (tomox *TomoX) ProcessCancelOrder(tomoXstatedb *tomox_state.TomoXStateDB, statedb *state.StateDB, chain consensus.ChainContext, coinbase common.Address, orderBook common.Hash, order *tomox_state.OrderItem) (error, bool) {
+	exRelayerFee := tomox_state.GetExRelayerFee(order.ExchangeAddress, statedb)
```

Comment:
The following function `GetExRelayerFee` doesn't return remaining fee of relayer.
It returns feeRate decimal. 
For example:
```
 _fee = 1 means feeRate = 1/1000 = 0.1%
```
To get remaining fee, let's get value at slot `_deposit`, then substract ` lockedFund = 20 000 TOMO`
```
func GetExRelayerFee(relayer common.Address, statedb *state.StateDB) *big.Int {
	slot := RelayerMappingSlot["RELAYER_LIST"]
	locBig := GetLocMappingAtKey(relayer.Hash(), slot)
	locBig = locBig.Add(locBig, RelayerStructMappingSlot["_fee"])
	locHash := common.BigToHash(locBig)
	return statedb.GetState(common.HexToAddress(common.RelayerRegistrationSMC), locHash).Big()
}
```